Uno.Gallery/Uno.Gallery.UWP/Views/Styles/DataGrid.xaml

```diff
@@ -0,0 +1,968 @@
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="Auto"/>
+                            <ColumnDefinition Width="*"/>
+                        </Grid.ColumnDefinitions>
+
+                        <Rectangle x:Name="BackgroundRectangle" Grid.ColumnSpan="2" Fill="{ThemeResource SystemControlTransparentBrush}"/>
+                        <Rectangle x:Name="InvalidVisualElement" Grid.ColumnSpan="2" Opacity="0" Fill="{ThemeResource DataGridRowInvalidBrush}"/>
+
+                        <localprimitives:DataGridRowHeader x:Name="RowHeader" Grid.RowSpan="3" localprimitives:DataGridFrozenGrid.IsFrozen="True"/>
+                        <localprimitives:DataGridCellsPresenter x:Name="CellsPresenter" Grid.Column="1"
+                            localprimitives:DataGridFrozenGrid.IsFrozen="True" MinHeight="32"
+                            AutomationProperties.AccessibilityView="Raw"/>
+                        <localprimitives:DataGridDetailsPresenter x:Name="DetailsPresenter" Grid.Row="1" Grid.Column="1"
+                            Background="{ThemeResource DataGridDetailsPresenterBackgroundBrush}"
+                            AutomationProperties.AccessibilityView="Raw"/>
+                        <Rectangle x:Name="BottomGridLine" Grid.Row="2" Grid.Column="1" HorizontalAlignment="Stretch" Height="1"/>
+                    </localprimitives:DataGridFrozenGrid>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+    </Style>
+
+    <Style TargetType="localprimitives:DataGridRowHeader">
+        <Setter Property="Background" Value="{ThemeResource DataGridRowHeaderBackgroundBrush}"/>
+        <Setter Property="IsTabStop" Value="False"/>
+        <Setter Property="SeparatorBrush" Value="{ThemeResource GridLinesBrush}"/>
+        <Setter Property="SeparatorVisibility" Value="Collapsed"/>
+        <Setter Property="AutomationProperties.AccessibilityView" Value="Raw"/>
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="localprimitives:DataGridRowHeader">
+                    <Grid x:Name="RowHeaderRoot">
+                        <VisualStateManager.VisualStateGroups>
+                            <VisualStateGroup x:Name="CommonStates">
+                                <VisualState x:Name="Normal"/>
+                                <VisualState x:Name="NormalCurrentRow">
+                                    <!-- For optional row currency and editing glyphs
+                                    <Storyboard>
+                                        <DoubleAnimation Storyboard.TargetName="CurrentRowGlyph" Storyboard.TargetProperty="Opacity" Duration="0" To="1"/>
+                                        <DoubleAnimation Storyboard.TargetName="EditingRowGlyph" Storyboard.TargetProperty="Opacity" Duration="0" To="0"/>
+                                    </Storyboard>
+                                    -->
```

Comment:
commented for which reason?

Reply:
Also same issue https://github.com/unoplatform/Uno.Gallery/pull/705#discussion_r1196056414